### PR TITLE
Fixing some TODOs in the intro

### PR DESCRIPTION
**Pull request by jbieniosek**

### CODE REVIEW

File: 01-intro-to-python/04-lists.md [hunk 1]
```diff
@@ -15,22 +15,58 @@ At the end of this lesson students will be able to:
...
+# chocolate
+# caramel
+
+```
+
+* **index**: The position of an element in a _sequence type_, that is, a type that stores data in order.  In Python, strings and lists are examples of sequence types.  In Python indexes start at 0.
+
+* **package**: A collection of _modules_ contained in a directory
+
+* **module**: A file containing python code.
```
Comment (CheezItMan) [inline 01-intro-to-python/04-lists.md:63]: Going with the above

```suggestion
```

**Review by CheezItMan — CHANGES_REQUESTED** [review]
I had a few suggestions involving putting vocab into tables.

File: 01-intro-to-python/04-lists.md [hunk 2]
```diff
@@ -15,22 +15,58 @@ At the end of this lesson students will be able to:
...
 
 ## Vocabulary
 
-__TODO__
-* list
-* index
-* package
+* **data structure**: A collection of data that follows a set of rules for modifying and accessing the data.  Python provides four built in data structures, _lists_, _dictionaries_, _tuples_ and _sets_.  This curriculum will cover lists and dictionaries.
+
+* **list**: An ordered collection of data where each element in the list can be accessed using it's index.  List indexes begin at 0.
```
[the remark this answers is not in the document]
Comment (kaidamasaki) [reply 01-intro-to-python/04-lists.md:28]: For "package" I'd probably say: "A collection of _modules_ distributed together" and list "egg" as a synonym.

[the remark this answers is not in the document]
Comment (CheezItMan) [reply 01-intro-to-python/04-lists.md:28]: Revised

File: 01-intro-to-python/04-lists.md [hunk 3]
```diff
@@ -15,22 +15,58 @@ At the end of this lesson students will be able to:
 
 ### Textbook for this section: [link to ada built lists]
```
Comment (CheezItMan) [inline 01-intro-to-python/04-lists.md:16]:
```suggestion
### [Textbook for this section](https://colab.research.google.com/drive/1TK9Enhh0mITZ1649l-r4_gzeg2B3eRRu?usp=sharing)
```

**Pull request merged**